reactioncommerce/reaction
# [WIP] 2014 Change Delete Product Tooltip To Archive Product

Resolves
#2014 

**Manual Test**
- Login as admin
- Click on a product
- Hover for 3 seconds on the archive icon on the selected panel header by the right part of your screen
- The tooltip should show `Archive product.`
- Also, click on the product to view the details.
- Click the pencil icon on a variant to view the details.
- On the panel header you would see an archive icon, with a tooltip saying `Archive Variant`
- You can also click a variant option to view its details.
- Below the edit form, you would see an archive icon with a tooltip saying `Archive Option`
